test/integration/dialects/sqlite/dao.test.js

```diff
@@ -3,6 +3,7 @@
 var chai = require('chai')
   , expect = chai.expect
   , Support = require(__dirname + '/../../support')
+  , Sequelize = Support.Sequelize
```

Comment:
``json(...)`` is static method inside ``Sequelize`` class, but ``this.sequelize`` was an instance of Sequelize. (it causes test failure.)

So, I added ``Sequelize`` from ``Support.Sequelize``